code/modules/antagonists/voidwalker/voidwalker_traumas.dm

```diff
@@ -51,12 +51,13 @@
 
 	for(var/obj/item/bodypart/bodypart as anything in owner.bodyparts)
 		untexture_limb(owner, bodypart)
+	owner.update_body()
 
 /// Apply the space texture
 /datum/brain_trauma/voided/proc/texture_limb(atom/source, obj/item/bodypart/limb)
 	SIGNAL_HANDLER
 
-	limb.add_bodypart_overlay(new bodypart_overlay_type)
+	limb.add_bodypart_overlay(new bodypart_overlay_type(), update = FALSE)
```

Comment:
I don't see any other update calls later down this proc

Reply:
No need for this, called later down the line in parent procs or by attachment/dismemberment